app/views/reports/index.html.erb

```diff
@@ -63,8 +63,19 @@
             <% end %>
           <% end %>
         </li>
+        <li>
+          <%= form_with url: placement_reports_path(format: :csv), method: :get do |f| %>
+            <%= button_tag(
+                  data: { disable_with: "Downloading Placements Report" },
+                  class: "btn-sm main-btn primary-btn btn-hover report-form-submit"
+                ) do %>
+              <i class="lni lni-check-box mr-10"></i> Placements Report
+            <% end %>
+          <% end %>
+        </li>
       </ul>
     </div>
+  </div>
```

Comment:
Was there a closing div missing from before? This looks like a bonus div closure.

Reply:
I had removed a UL closing tag that wasn't necessary, I think this may be a bonus